pkg/api/server/v1alpha2/results_test.go

```diff
@@ -102,6 +104,69 @@ func TestCreateResult(t *testing.T) {
 	}
 }
 
+func TestUpdateResult(t *testing.T) {
+	srv, err := New(test.NewDB(t))
+	if err != nil {
+		t.Fatalf("failed to create server: %v", err)
+	}
+	ctx := context.Background()
+
+	tt := []struct {
+		name    string
+		create  *pb.Result
+		update  *pb.Result
+		expect  *pb.Result
+		errcode codes.Code
+	}{
+		{
+			name:   "test success",
+			update: &pb.Result{Annotations: map[string]string{"foo": "bar"}},
+			expect: &pb.Result{Annotations: map[string]string{"foo": "bar"}},
+		},
+		{
+			name:   "test update with empty result",
+			expect: &pb.Result{},
+		},
+	}
```

Comment:
Other good things to test are all of the conditions that return errors.

Reply:
Done